tedana/utils.py

```diff
@@ -384,3 +385,54 @@ def get_resource_path():
     Based on function by Yaroslav Halchenko used in Neurosynth Python package.
     """
     return op.abspath(op.join(op.dirname(__file__), "resources") + op.sep)
+
+
+def get_system_info():
+    """Return information about the system tedana is being run on.
+
+    Returns
+    -------
+    dict
+        Info about system where tedana is run on.
+    """
+    system_info = platform.uname()
+
+    system_dict = {
+        "System": system_info.system,
+        "Node": system_info.node,
+        "Release": system_info.release,
+        "Version": system_info.version,
+        "Machine": system_info.machine,
+        "Processor": system_info.processor,
+    }
+
+    return system_dict
+
+
+def write_tedana_command(arg_vals):
```

Comment:
Following up from https://github.com/eurunuela/pySPFM/pull/28- I think it would be better if we could use the CLI commands directly, and _not_ compile a tedana command when users don't use the CLI. WDYT of swapping out this function with the changes I proposed in pySPFM?